github/Team.py

```diff
@@ -218,6 +219,7 @@ def add_membership(self, member: NamedUser, role: Opt[str] = NotSet) -> None:
         headers, data = self._requester.requestJsonAndCheck(
             "PUT", f"{self.url}/memberships/{member._identity}", input=put_parameters
         )
+        return github.Membership.Membership(self._requester, headers, data, completed=True)
```

Comment:
Happy to approve this change, but can you extend the tests in `tests/Team.py`? Tests already call `add_membership`, so simply assert some attributes of the returned `Membership` instance.